### ISSUE TITLE
DynamicPage: Divider bar doesn't disappear when no navigationItems are passed

### ISSUE THREAD

**Issue by charlie-maloney** [SAP/ui5-webcomponents-react#1152]
**Describe the bug**
When we don't pass any navigation item props, the divider bar doesn't disappear
Fiori: https://experience.sap.com/fiori-design-web/dynamic-page-layout/

**Isolated Example**
https://codesandbox.io/s/stoic-ellis-shqip?file=/src/App.js:1903-2269

**To Reproduce**
Please see code sandbox

**Expected behavior**
We expected the divider bar to go away

**Screenshots**
![Screen Shot 2021-01-12 at 3 27 37 PM](https://user-images.githubusercontent.com/22550767/104386876-d5bdd380-54ea-11eb-82f1-53aa5d6f83ca.png)

**UI5 Web Components for React Information**

![Screen Shot 2021-01-12 at 3 29 53 PM](https://user-images.githubusercontent.com/22550767/104386974-09006280-54eb-11eb-8f3a-9947079ba586.png)

Operating System: Mac
Browser: Chrome

Thank you!

**Comment by MarcusNotheis** [comment]
Hey @charlie-maloney,
thanks for reporting! We have fixed this issue in our latest release (0.13.1), so if you update your dependencies to this versions it should work fine:
```json
  "dependencies": {
    "@ui5/webcomponents": "1.0.0-rc.11",
    "@ui5/webcomponents-base": "0.28.0",
    "@ui5/webcomponents-fiori": "1.0.0-rc.11",
    "@ui5/webcomponents-icons": "1.0.0-rc.11",
    "@ui5/webcomponents-react": "0.13.1",
  },
```

You can find an updated [codesandbox here](https://codesandbox.io/s/stoic-feynman-2snwk?file=/src/App.js).

**Comment by charlie-maloney** [comment]
Hey @MarcusNotheis -- amazing! thanks for letting me know!
